webapps/frontend/app/viewmodels/containers/albabackend.js

```diff
@@ -153,7 +151,8 @@ define([
         };
         self.load = function(contents) {
             if (contents === undefined) {
-                contents = '_dynamics,-statistics,-ns_data,_relations';
+                // TODO: Remove collecting all dynamics and all relations on every load action
+                contents = '_dynamics,-statistics,-ns_data,-local_stack,_relations';
```

Comment:
Not collecting local_stack anymore, does this give the desired results in all pages making use of ALBA Backends?
Eg: vpool creation, vpool extension, backend detail page, linking backends, ...

Reply:
Local stack is nowhere used directly. All computed values that use the local stack are however still as important.